server/src/main/java/org/opensearch/extensions/rest/RestSendToExtensionAction.java

```diff
@@ -105,12 +108,16 @@ public List<Route> routes() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
```

Comment:
qq, out of scope of this PR, should we change the method name to `handleRequest` in TransportRequest.java at some point in future?